loki/visitors/pretty_visualize.py

```diff
@@ -0,0 +1,349 @@
+# (C) Copyright 2018- ECMWF.
+# This software is licensed under the terms of the Apache Licence Version 2.0
+# which can be obtained at http://www.apache.org/licenses/LICENSE-2.0.
+# In applying this licence, ECMWF does not waive the privileges and immunities
+# granted to it by virtue of its status as an intergovernmental organisation
+# nor does it submit to any jurisdiction.
+
+"""
+Pretty-Graph-Visualizer classes for IR
+"""
+from itertools import chain
+from codetiming import Timer
+from graphviz import Digraph
```

Comment:
This import should be guarded against missing graphviz, similar to here:
https://github.com/ecmwf-ifs/loki/blob/75fc36f389b560cbf6f18ee22c80a80860f36ac8/loki/frontend/ofp.py#L15-L21

Reply:
Fix with 781ba3d2635147668e8770f119af5d772b95bf0d